2_process.R

```diff
@@ -49,19 +49,69 @@ p2_targets <- list(
     pattern = map(p2_huc8s_coords)
   ),
   
+  
+  # Prepare HUC 4s for SVG
+  tar_target(
+    p1_huc4s_simp_sf,
+    p1_huc4s_sf %>% 
+      st_intersection(p1_conus_sf) %>% 
+      rmapshaper::ms_simplify(0.01)
```

Comment:
I had to add a `st_buffer(0)` before `st_intersection` to bypass a self-intersection topology error

Reply:
Should I add that in here? I didn't have to do that :(